Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -81,6 +81,29 @@ class VisualRecognitionTests: XCTestCase {
         instantiateVisualRecognition()
     }
 
+    /** Teardown logic to cleanup dangling resources after all the tests have run */
+    override func tearDown() {
+        let teardownExpectation = self.expectation(description: "Teardown processing after all tests.")
+        visualRecognition.listClassifiers(verbose: true, failure: failWithError) { classifiers in
+            let classifiersToDelete = classifiers.classifiers.filter { $0.name.starts(with: "swift-sdk-unit-test-") }
+            if classifiersToDelete.count > 0 {
+                // allow zip files to propagate through object storage, so that
+                // they will be deleted when the service deletes the classifier
+                // (otherwise they remain and dramatically slow down the tests)
+                sleep(15) // wait 15 seconds
```

Comment:
Are we waiting for the classifier to be `ready` or `failed` (a short time after, hah)? I think you might be right when you said we should move away from deleting in the tests and do a purge every week or so.